docs/modernization/WebViewCompatible.md

```diff
@@ -0,0 +1,224 @@
+
+## Prerequisites
+
+:heavy_check_mark: Visual Studio 2017.
+
+:heavy_check_mark: Windows 10 Insider Preview Build 17110 or a later release for WebViewControl-based underlying implementation.
+
+:heavy_check_mark: .NET Framework 4.6.2 or a later release.
+
+:heavy_check_mark: Configure your application for high DPI support. To learn how, see [this section](#high-dpi) of the guide.
+
+## Feature limitations
+
+When compared to the UWP **WebView** control, the current release of the WPF and Windows Forms **WebView** controls has some limitations. For the complete list of these limitations, see [Known Issues of the WebView control for Windows Forms and WPF applications](WebView-known-issues.md).  The WebViewCompatible control has further limitations because it exposes a common interface that is a subset of common functionality across the two underlying implementations.  In a system that supports the Windows 10 WebViewControl, this control will exhibit those limitations plus further exclusions to the exposed functionality. In a legacy environment, this control will use WebBrowser, and have the same additional interface limitations.
+
+## Add the WebViewCompatible control to the Visual Studio Toolbox
+
+First, open the Visual Studio **Toolbox**, then right-click anywhere in the toolbox, and select the **Choose Items** option.
+
+<!-- Remove these steps before going public. This is for early adopters only -->
+
+### For Windows Forms applications
+
+1. In the **.NET Framework Components** tab of the **Choose Toolbox Items** dialog box.
+
+2. Use the **Browse** button to locate the **Microsoft.Toolkit.Win32.UI.Controls.dll** in your NuGet package folder.
+
+   For help finding that folder, see [Managing the global packages, cache, and temp folders](https://docs.microsoft.com/nuget/consume-packages/managing-the-global-packages-and-cache-folders).
+
+3. Add that file to the list of Toolbox controls, and then close the **Choose Toolbox Items** dialog box.
+
+   The **WebView** control appears in the **All Windows Forms** section of the **Toolbox**.
+
+   In **Solution Explorer**, the **Microsoft.Toolkit.Win32.UI.Controls.dll** file appears in the **References** list.
+
+### For WPF applications
+
+The **WebViewCompatible** control appears in the **Windows Community Toolkit** section of the **Toolbox** in Visual Studio or Blend and you can drag it directly the designer. You can also create an instance of the **WebViewCompatible** control in code, but we recommend that you do not add **WebViewCompatible** controls to popup windows because support for that scenario will soon be disabled for security reasons.
+
+<a id="high-dpi" />
```

Comment:
There's a lot of duplication between the original WebView documentation for the toolkit and this. Should be refactored so we only need to maintain one place